src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment(this.state) }>
                     +
                 </button>
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.decrement(this.state) }>
```

Comment:
Why pass anything to these functions? They seem to not take any parameters. Try just passing their definitions instead:
```javascript
<button onClick={this.props.decrement}> 
```